src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/index.unit.spec.jsx

```diff
@@ -1,280 +1,264 @@
 import React from 'react';
 import { expect } from 'chai';
-import sinon from 'sinon';
-import { mount, shallow } from 'enzyme';
 import moment from 'moment';
-
 import {
-  DateTimeSelectPage,
-  getOptionsByDate,
-} from '../../../../new-appointment/components/DateTimeSelectPage';
-import { FETCH_STATUS } from '../../../../utils/constants';
+  createTestStore,
+  renderWithStoreAndRouter,
+} from '../../../mocks/setup';
+import userEvent from '@testing-library/user-event';
 
-const availableDates = ['2019-10-29'];
-const availableSlots = [
-  {
-    start: '2019-10-29T09:30:00',
-    end: '2019-10-29T09:50:00',
-  },
-  {
-    start: '2019-10-29T10:00:00',
-    end: '2019-10-29T10:20:00',
-  },
-];
+import DateTimeSelectPage from '../../../../new-appointment/components/DateTimeSelectPage';
+import { FETCH_STATUS } from '../../../../utils/constants';
+import { waitFor } from '@testing-library/dom';
+import { Route } from 'react-router-dom';
+import parentFacilities from '../../../../services/mocks/var/facilities.json';
+import { transformParentFacilities } from '../../../../services/organization/transformers';
+
+function getMondayTruFriday(date) {
+  if (date.day() === 6) {
+    // Move date to next week Monday
+    return date.day(8);
+  } else if (date.day() === 0) {
+    // Move date to current week Monday
+    return date.day(1);
+  }
+  return date;
+}
+
+function getAppointmentTimeSlots(date, count) {
+  const start = moment(date)
+    .minute(0)
+    .second(0);
+  const end = moment(date)
+    .minute(15)
+    .second(0);
+  const collection = [];
+
+  for (let i = 0; i < count; i++) {
+    collection.push({
+      start: start.clone().format('YYYY-MM-DDTHH:mm'),
+      end: end.clone().format('YYYY-MM-DDTHH:mm'),
+    });
+    start.minute(start.minute() + 15);
+    end.minute(end.minute() + 15);
+  }
+  return collection;
+}
+
+const parentFacilitiesParsed = transformParentFacilities(
+  parentFacilities.data.map(item => ({
+    ...item.attributes,
+    id: item.id,
+  })),
+);
+
+const availableSlots = getAppointmentTimeSlots(
+  getMondayTruFriday(moment().add(5, 'd')),
+  2,
+);
 
 describe('VAOS <DateTimeSelectPage>', () => {
```

Comment:
Can you move the test in [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/tests/new-appointment/components/DateTimeSelectPage/select-date-page.unit.spec.jsx) to this file, and remove that other? I don't think we need to have them in separate files, now that everything is using RTL.

And if you feel like it, you could update it to use the `ByRole` selectors more.